lib/http-server.js

```diff
@@ -76,6 +76,13 @@ function HttpServer(options) {
     } : null));
   }
 
+  if (options.extraHeaders) {
+    options.extraHeaders.forEach(function (header) {
+      var split = header.split(/:(.+)?/);
```

Comment:
```suggestion
      var split = header.split(':', 2);
```
might be a more obvious way to do this.

Reply:
colons are allowed in the value part of the header (e.g. `Content-Type: Foo: 2: Tokyo Drift` is valid), so we'd need to concat everything after the first match again anyway. I would have though of doing it this way first too, but the regex capture group actually ends up being more terse, which is neat

Reply:
@thornjad that's why I suggested passing `2` as 2nd argument to the `split(separator, limit)` syntax.

Reply:
```js
'Content-Type: Foo: 2: Tokyo Drift'.split(':', 2)
> Array [ "Content-Type", " Foo" ]
```

The limit just sets the max length of the resulting array

Reply:
:facepalm: you're right, sorry.

Reply:
No problem! The way you were thinking is the way it works in other languages like Python and Java, JS is the weird one out, as it so often is